core/server/worker/src/main/java/alluxio/worker/grpc/BlockReadHandler.java

```diff
@@ -515,7 +513,7 @@ protected DataBuffer getDataBuffer(BlockReadRequestContext context, long offset,
         blockReader = context.getBlockReader();
         Preconditions.checkState(blockReader != null);
         startTransferMs = System.currentTimeMillis();
-        if (IS_READER_BUFFER_POOLED) {
+        if (Configuration.getBoolean(PropertyKey.WORKER_NETWORK_READER_BUFFER_POOLED)) {
```

Comment:
is it possible to put it in the constructor?  because reading from configuration is relatively expensive

Reply:
Get value from Configuration is just get it from a map inside, so it is not really expensive I think.

If we put this to the constructor, it have no chance to refresh the cached value.